README.md

```diff
@@ -1,49 +1,75 @@
 # Single Sign On
 
-Webapp intended to be used as an iframe.
+This webapp works as a centralized storage for shared data between different decentraland applications.
 
-Contains the logic to Get, Store and Clear the user identity.
+It is intended to be instanced as an iframe inside the different dapps using the [@dcl/single-sign-on-client](https://github.com/decentraland/single-sign-on-client).
 
-The idea of this project is to have unified place in which the identity of the user can be stored.
+## V1
 
-This replaces the need of each application storing it by themselves, making the user have to sign a message when logging in on each app.
+Supports storing, clearing and obtaining the user identity via post messages.
 
-## Integration
+Version 0.1.0 or lower of the `@dcl/single-sign-on-client` has to be used to easily communicate with it.
 
-The simplest way to use it is by using the corresponding client in the application it needs to be integrated in.
+This version will not receive future updates. As version 2 is the recommended one.
 
-https://github.com/decentraland/single-sign-on-client <- Client Repo
+It is still currently supported by the SSO webapp to keep support applications that have not yet migrated to V2.
 
-Just check the README on that repo for instructions.
+Handled messages have to follow the schema:
```

Comment:
We don't have control over the handled messages, so `have to follow` which implies requirement might be better expressed as just `follow`.
```suggestion
Handled messages follow the schema:
```